Gemfile

```diff
@@ -12,7 +12,7 @@ group :development do
   gem 'berkshelf', '~> 3.0'
 
   # Use Test Kitchen with Vagrant for converging the build environment
-  gem 'test-kitchen',    '~> 1.4.0.rc.1'
-  gem 'kitchen-vagrant', '~> 0.17.0.rc.1'
+  gem 'test-kitchen',    '~> 1.4.0'
```

Comment:
Unrelated change. Perhaps do this in another PR. You should test that the things still work when doing this